utility/utils.py

```diff
@@ -1801,7 +1801,10 @@ def run_fio(**fio_args):
     log.debug(f"Config Recieved for fio: {fio_args}")
 
     if fio_args.get("filename"):
-        opt_args = f" --filename={fio_args['filename']}/file"
+        file_name = fio_args["filename"]
+        if os.path.isdir(file_name):
+            file_name = f"{file_name}/file"
+        opt_args = f" --filename={file_name}"
```

Comment:
Does it makes sense to add info on this into method doc ?

Reply:
Done.